CondTools/HLT/plugins/AlCaRecoTriggerBitsRcdUpdate.cc

```diff
@@ -200,7 +304,28 @@ void AlCaRecoTriggerBitsRcdUpdate::writeBitsToDB(const AlCaRecoTriggerBits &bits
     throw cms::Exception("NotAvailable") << "PoolDBOutputService not available.\n";
   }
 
-  poolDbService->writeOneIOV(bitsToWrite, firstRunIOV_, "AlCaRecoTriggerBitsRcd");
+  const std::string recordName("AlCaRecoTriggerBitsRcd");
+
+  // when updating existing tag, compare payload hashs and skip appending new hash if it's same with last iov's
+  poolDbService->startTransaction();
+  auto newHash = poolDbService->session().storePayload(bitsToWrite);
+  cond::TagInfo_t tag_info;
+
+  if (poolDbService->tagInfo(recordName, tag_info)) {
+    if (newHash != tag_info.lastInterval.payloadId) {
+      std::cout << "## Appending to existing tag..." << std::endl;
```

Comment:
can it be `LogPrint`?

Reply:
thanks, moved to `LogInfo` like the other parts